crates/oxc_linter/src/snapshots/prefer_hooks_in_order.snap

```diff
@@ -0,0 +1,409 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 216
+---
+  ⚠ eslint-plugin-jest(prefer-hooks-in-order): Prefer having hooks in a consistent order.
+   ╭─[prefer_hooks_in_order.tsx:6:21]
+ 5 │                         });
+ 6 │ ╭─▶                     beforeAll(() => {
+ 7 │ │                           createMyDatabase();
+ 8 │ ╰─▶                     });
+ 9 │                     };
+   ╰────
+  help: `"beforeAll"` hooks should be before any `"afterAll"` hooks
```

Comment:
Nitpick: Looks like should be: `help: "beforeAll" hooks should be before any "afterAll" hooks`

Reply:
Done.